caffe2/python/optimizer.py

```diff
@@ -17,6 +17,15 @@
 
 AuxOptimizerParams = namedtuple("AuxOptimizerParams", ["local", "shared"])
 
+def _get_lr_blob_name():
```

Comment:
That would work for your use-case, and produce correct code. But it also makes this change a no-op, and doesn't fix the performance problem. Would it be reasonable to add a flag like `unique_lr_per_param=False`?